README.md

```diff
@@ -462,6 +463,28 @@ to live.
 
 Note: `chronos-sync.rb` does not delete jobs by default. You can pass the `--delete-missing` flag to `chronos-sync.rb` to remove jobs. Alternatively, you can manually remove it using the API or web UI.
 
+## Mesos Framework Authentication
+
+To enable the credentials in Chronos:
+
+* Run mesos-master with the credentials flag and the path to file with authorized users + their secrets.
```

Comment:
Note that this --credentials file is for all frameworks and slaves registering with Mesos. In enterprise installations, the cluster admin will have already configured credentials in Mesos, so the user launching Chronos just needs to specify the principal+secret given to them by the cluster/security admin.
Might be better to lead with the Chronos flags and mention the Mesos flags after as a note to the administrator.